packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -367,23 +367,46 @@ const DataFilters = ({
                     {/* Autocomplete */}
                     {filterPopoverCurrentFilter?.variant === "autocomplete" &&
                       filterPopoverCurrentFilter?.options && (
-                        <Autocomplete
-                          label={filterPopoverCurrentFilter.label}
-                          value={
-                            (inputValues[
-                              filterPopoverCurrentFilter.id
-                            ] as string) ?? ""
-                          }
-                          onBlur={function ro() {}}
-                          onChange={function ro() {}}
-                          onFocus={function ro() {}}
-                          onInputChange={function ro() {}}
-                          options={filterPopoverCurrentFilter.options.map(
-                            (option: { label: string }) => ({
-                              label: option.label,
-                            })
-                          )}
-                        />
+                        <Box
+                          sx={{
+                            display: "flex",
+                            gap: 2,
+                            alignItems: "flex-end",
+                          }}
+                        >
+                          <Box sx={{ width: "100%" }}>
+                            <Autocomplete
+                              label={filterPopoverCurrentFilter.label}
+                              value={
+                                (inputValues[
+                                  filterPopoverCurrentFilter.id
+                                ] as string) ?? ""
+                              }
+                              onChange={(_, value) => {
+                                const label =
```

Comment:
@benlister-okta Can we pull this logic up into the `handleInputChange`?

Reply:
Never mind, don't think we can :( How about making another function like... `getAutoCompleteLabel` or something like that? Generally, I just try to avoid having to much logic inside JSX

Reply:
@bryancunningham-okta I pulled back my last change and went with your idea of moving the logic to a `getAutoCompleteLabel` function. Looking at the changes I had in `handleInputChange`, it felt like it was getting too complicated overall. Let me know what you think of this approach or if you think and could be further optimized.